content/en/docs/setup/install/external-controlplane/index.md

```diff
@@ -200,6 +200,11 @@ and installing the sidecar injector webhook configuration on the remote cluster
     EOF
     {{< /text >}}
 
+    {{< tip >}}
+    If your cluster name contains `/` (slash) characters, replace them with `--slash--` in the `injectionURL`,
+    e.g., `injectionURL: https://1.2.3.4:15017/inject/cluster/cluster--slash--1/net/network1`.
+    {{< /tip >}}
+
```

Comment:
My only comment is I didn't read this close enough to see just change the slashes in the cluster name, so good to have an example. Should we highlight `cluster name` more? Maybe we can't in a tip.

Reply:
Good idea, I highlighted the "cluster--slash--1" part in the URL.